src/public/drug/sections/LinkedDiseases/Section.js

```diff
@@ -15,7 +15,7 @@ const columns = [
     renderCell: d =>
       d.therapeuticAreas.map((t, i) => (
         <React.Fragment key={i}>
-          {i > 0 ? ', ' : null}
+          {i > 0 ? <br /> : null}
```

Comment:
Could you try this?

```js
d.therapeuticAreas.map((t, i) => (
  <div key={i}>
    <Link to={`/disease/${t.id}`}>{t.name}</Link>
  </div>
))
```
This way, we don't rely on `<br>` for layout and we can get rid of the tertiary operator.